### DISCUSSION TITLE
YOLO badge not showing on website

### DISCUSSION THREAD

**Discussion by vivektripaathi** [community/community#109547] [Other Feature Feedback, Questions, & Ideas]
### Question

YOLO badge not showing on website. Is it a bug or do i have to make some changes in my account?

### Body

Either only three badge showing on my profile or YOLO badge is not showing.
- Today i found out that i got YOLO badge on my profile in github ios app but the same is not showing on github webpage.
- YOLO bage is showing in [achievements](https://github.com/vivek-tripathi-9005?tab=achievements) section of my account but not on [profile](https://github.com/vivek-tripathi-9005).

**Comment by queenofcorgis** [comment]
Hi @vivek-tripathi-9005 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
